variables.tf

```diff
@@ -217,3 +213,9 @@ variable "password_policy" {
   default     = {}
   description = "Allows to create/set aws iam users password policy for better security"
 }
+
+variable "security_hub_name" {
```

Comment:
why to have separate alone config for security hub name?
we already have  security hub related bulk configs under var.alarm_actions.security_hub_alarms object variable and seems we can place name there